Source/Basic Shapes/SvgImage.cs

```diff
@@ -240,78 +224,50 @@ protected override void Render(ISvgRenderer renderer)
                         renderer.PopBoundable();
                     }
 
-
-                    this.ResetClip(renderer);
-                    this.PopTransforms(renderer);
+                    ResetClip(renderer);
+                    PopTransforms(renderer);
+                }
+                finally
+                {
+                    if (bmp != null)
+                        bmp.Dispose();
                 }
-                // TODO: cache images... will need a shared context for this
             }
+            // TODO: cache images... will need a shared context for this
         }
 
-        public object GetImage()
+        protected object GetImage(string uriString)
         {
-            return this.GetImage(this.Href);
-        }
-
-        public object GetImage(string uriString)
-        {
-            string safeUriString;
-            if (uriString.Length > 65519)
-            {
-                //Uri MaxLength is 65519 (https://msdn.microsoft.com/en-us/library/z6c2z492.aspx)
-                safeUriString = uriString.Substring(0, 65519);
-            }
-            else
-            {
-                safeUriString = uriString;
-            }
+            // Uri MaxLength is 65519 (https://msdn.microsoft.com/en-us/library/z6c2z492.aspx)
+            // main use is data URI scheme, but very long to use it as data URI scheme.

```

Comment:
I overlooked this one - could you please rephrase this in the other PR? I'm not sure I understand the meaning, but I guess it means that a very long URI may happen if using the data URI schema?

Reply:
@mrbean-bremen 

Thanks for your review.
That's right.